### PR TITLE
Use DashedRoute for examples, fix typo

### PR DESCRIPTION
**Pull request by bcrowe**
Follows https://github.com/cakephp/app/pull/245

### CODE REVIEW

**Comment by dereuromark** [comment]
Funny thing that I also did just start on it ;) https://github.com/cakephp/docs/pull/2723

**Comment by dereuromark** [comment]
Feel free to merge mine or what is useful of it into yours and close mine then.

**Comment by bcrowe** [comment]
:sob:  Haha, sure thing!

File: en/appendices/3-1-migration-guide.rst [hunk 1]
```diff
@@ -4,6 +4,13 @@
 CakePHP 3.1 is a fully API compatible upgrade from 3.0. This page outlines
 the changes and improvements made in 3.1.
 
+Routing
+=======
+
+The default route class has been changed to `DashedRoute` in the ``cakephp/app``
```
Comment (markstory) [inline en/appendices/3-1-migration-guide.rst]: Need double backticks for preformatted code.

Comment (bcrowe) [inline en/appendices/3-1-migration-guide.rst]: :+1: Done

**Pull request merged**